hugo-site/content/docs/virtualbox-create-vm.md

```diff
@@ -0,0 +1,129 @@
++++
+title = "Installing: Creating a VirtualBox Virtual Machine"
+MenuTitle = "VirtualBox VMs"
+Description = "Simple instructions to create and configure a virtual machine for Project Trident."
+chapter = false
+weight = 2
+pre = "<i class='fas fa-info'></i>   "
++++
+
```

Comment:
Might be a good idea to put a quick note/table at the top with the key points for running PT as a guest VM:
OS Type: BSD / FreeBSD (64-bit)
Memory: 2048MB (minimum), 4096MB (recommended)
Disk Space: 15GB (minimum), 20GB (recommended)
CPUs: 1+